tslint.json

```diff
@@ -1,14 +1,21 @@
 {
   "rulesDirectory": [
-    "node_modules/codelyzer"
+    "codelyzer"
```

Comment:
This change fixes the issue of requiring SPAs to have their own version of codelyzer listed in package.json. "These values are handled using node module resolution semantics..."